autoload/youcompleteme.vim

```diff
@@ -223,7 +223,7 @@ function! youcompleteme#EnableCursorMovedAutocommands()
     autocmd!
     autocmd CursorMoved * call s:OnCursorMovedNormalMode()
     autocmd CursorMovedI * let s:current_cursor_position = getpos( '.' )
```

Comment:
Why doesn't this call s:OnInsertEnter as well? I mean, I get that it is not for the InsertEnter autocommand, but it does the same thing. Why is the line 226 extracted into a function, but line 225 isn't?